src/components/activity-header/nav-pages.tsx

```diff
@@ -36,16 +52,11 @@ export class NavPages extends React.PureComponent <IProps> {
     );
   }
 
-  private renderPageButton = (page: Page, index: number) => {
-    const currentClass = this.props.currentPage === (index + 1) ? "current" : "";
-    return (
-      <button className={`page-button ${currentClass}`} key={`index ${index}`} onClick={this.handleButtonClick(index + 1)}>
-        <div className={`label ${currentClass}`}>{(index + 1).toString()}</div>
-      </button>
-    );
+  private handlePaginate = (page: any) => {
+    if (page.selected >= 0) {
+      this.props.onPageChange(page.selected + 1);
+    } else {
+      this.props.onPageChange(0);
+    }
```

Comment:
I think you can simplify the contents of this function to:
```
    const { onPageChange } = this.props;
    page.selected !== undefined ? onPageChange(page.selected + 1) : onPageChange(0);
```

Reply:
or even:
```
this.props.onPageChange(pageSelected != null ? page.selected + 1 : 0);
```

Reply:
One small tweak on Kirk's version:
`this.props.onPageChange(page.selected != null ? page.selected + 1 : 0);`

Reply:
Do what I mean, not what I say... 😉

Reply:
👍